fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/directory/AgilityContextTest.java

```diff
@@ -707,12 +707,13 @@ void testAutoCommitVersionStampInnerSleepUseApply() {
     }
 
 
-    @Test
-    void luceneTransactionPriorityVerificationTest() {
+    @ParameterizedTest
+    @BooleanSource
+    void luceneTransactionPriorityVerificationTest(boolean usePriorityBatch) {
         // Assert the expected priorities for user context and agility context
         try (FDBRecordContext userContext = openContext()) {
             final FDBTransactionPriority userPriority = userContext.getPriority();
```

Comment:
Isn't `userPriority` always going to be `DEFAULT`?
I don't think there's much risk of messing this up, but the goal is that when you create an `AgileContext` with a callerContext that has `BATCH` priority you can upgrade to `DEFAULT`. That scenario is not covered here.